Issues/168 (#303)
V4Fire/Client: src/core/component/directives/image/README.md

# core/component/directives/image

> This directive provides API to set a `background-image` or an `src` to nodes.

Reviewer: ```suggestion
This directive provides API to load images by using `background-image` or `src`.
```

## Usage

```

> < .&__not-img v-image = {

Reviewer 2: амперсанд `&` не забыл?
